pages/desktop/details.py

```diff
@@ -69,11 +69,6 @@ class Details(Base):
     _reviews_locator = (By.CSS_SELECTOR, "section#reviews div")
     _add_review_link_locator = (By.ID, "add-review")
 
-    _add_to_collection_locator = (By.CSS_SELECTOR, ".collection-add.widget.collection")
-    _add_to_collection_widget_button_locator = (By.CSS_SELECTOR, ".collection-add-login .register-button .button")
-    _add_to_collection_widget_login_link_locator = (By.CSS_SELECTOR, "div.collection-add-login p:nth-child(3) > a")
```

Comment:
This is referenced in `collection_widget_login_link`, which should also be removed.